lib/twilio.rb

```diff
@@ -0,0 +1,10 @@
+require 'twilio-ruby'
+
+# put your own credentials here
+account_sid = ENV['ACCOUNT_SID']
+auth_token =  ENV['AUTH_TOKEN']
+
+# set up a client to talk to the Twilio REST API
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+@message = @client.account.messages.create({:to => "+447477974890",:from => "+447477974890",:body => "Hello!"})
```

Comment:
Line is too long. [111/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.